[CQA] Shorten abstracts in Provisioning Hosts (#4954)
theforeman/foreman-documentation: guides/common/modules/proc_creating-discovery-rules-by-using-web-ui.adoc

= Creating Discovery rules by using {ProjectWebUI}

[role="_abstract"]
You can automate provisioning of discovered hosts by creating Discovery rules that assign hosts to host groups based on hardware attributes.

> For example, you can automatically provision hosts with high CPU counts as hypervisors or hosts with large disks as storage servers.

Reviewer: same as for CLI.

Author: ```suggestion
For example, you can automatically provision hosts with a large number of CPU cores as hypervisors and hosts with large disks as storage servers.
```
